lib/LaTeXML/resources/RelaxNG/LaTeXML-structure.rnc

```diff
@@ -21,7 +21,7 @@ element document { document_attributes, document_model }
 
 ## The content allowable as the main body of the document.
 document.body.class = Para.model | paragraph | subsubsection
-  | subsection | section | chapter | part | slide | sidebar
+  | subsection | section | chapter | part | slideorsequence | sidebar
```

Comment:
No, I meant literally `slide | slidesequence` here. Defining the pattern is perfectly legit rng, but violates the unstated naming convention of `*.class` and such which greatly helps documentation. In fact, I would have used a class here, to avoid the repetition, but for the slowly accumulation of section, then subsection, then...